layout/ut/plan.c

```diff
@@ -70,6 +73,7 @@ static int lap_ut_server_start(void)
 static void lap_ut_server_stop(void)
 {
 	m0_rpc_server_stop(&lap_ut_sctx);
+	lap_ut_sctx = lap_ut_sctx0;
```

Comment:
Is this a common practice in our UTs to save and restore `m0_rpc_server_ctx` like this?

Reply:
I don't know, I haven't seen other UTs failing this way.